extension/llm/export/partitioner_lib.py

```diff
@@ -142,6 +142,12 @@ def _validate_ios_version() -> None:
     return CoreMLPartitioner(  # pyre-fixme[16]
         compile_specs=compile_specs,
         take_over_mutable_buffer=take_over_mutable_buffer,
+        # Skip embedding ops in coreml delegation due to
```

Comment:
Is it only applicable to small models or also large models like 7b?

Reply:
I saw same error on llama 1B

Reply:
Right I understand - but this is supposed to be working for 7b

Reply:
I just want to check if this pr will break the claim in https://machinelearning.apple.com/research/core-ml-on-device-llama

Reply:
They currently link to 0.4 branch in the blog post, maybe it's okay, but still not great if we break it

Reply:
I can check the claim.  But which claim in https://machinelearning.apple.com/research/core-ml-on-device-llama specifically do we want to test?

The article is mostly about direct use of the coremltools API and says very little about ExecuTorch, except for one paragraph "Model Export via torch.export and Inference via ExecuTorch" that is light on details.

Reply:
They link to executorch page https://github.com/pytorch/executorch/tree/v0.4.0/examples/models/llama2#step-4-run-on-your-computer-to-validate with some instructions. But anyway, let me tag @YifanShenSZ on this PR